app/forms/schools/add_participant_form.rb

```diff
@@ -322,7 +322,7 @@ def store_validation_result!(profile)
     end
 
     def display_name
-      type == :self ? "your" : "#{full_name&.titleize}’s"
+      type == :self ? "your" : "#{full_name}’s"
```

Comment:
Should this use `possessive_name` ?

Reply:
I left this one as we can't see helpers from the form object and moving the logic would be a bit more involved. I agree it should though.